docs/user/security/securing-communications/elasticsearch-mutual-tls.asciidoc

```diff
@@ -5,70 +5,89 @@
-client. In a mutual TLS configuration, the client also presents a signed certificate to authenticate itself to the server.
+Secure Sockets Layer (SSL) and Transport Layer Security (TLS) provide encryption for data-in-transit. While these terms are often used
+interchangeably, {kib} only supports TLS, which supersedes the old SSL protocols.
 
-When {security} is enabled on your cluster, each request that {kib} makes to {es} must be authenticated. Most requests made through {kib} to
-{es} are authenticated by using the credentials of the logged-in user. There are, however, a few internal requests that the {kib} server
-needs to make to the {es} cluster. For this reason, you must configure credentials for the {kib} server to use for those requests.
+TLS requires X.509 certificates to authenticate the communicating parties and perform encryption of data-in-transit. Each certificate
+contains a public key and has and an associated -- but separate -- private key; these keys are used for cryptographic operations. {kib}
+supports certificates and private keys in PEM or PKCS#12 format.
+
+In a standard TLS configuration, the server presents a signed certificate to authenticate itself to the client. In a mutual TLS
+configuration, the client also presents a signed certificate to authenticate itself to the server.
+
+When {security} is enabled on your cluster, each request that {kib} (the client) makes to {es} (the server) must be authenticated. Most
+requests made by end users through {kib} to {es} are authenticated by using the credentials of the logged-in user. There are, however, a few
+internal requests that {kib} needs to make to {es}. For this reason, you must configure credentials for {kib} to use for those requests.
 
 If {kib} has `elasticsearch.username` and `elasticsearch.password` configured, it will attempt to use these to authenticate to {es} via the
 {ref}/native-realm.html[Native realm]. However, {kib} also supports mutual TLS authentication with {es} via a {ref}/pki-realm.html[Public
 Key Infrastructure (PKI) realm]. To do so, {es} needs to verify the signature on the {kib} client certificate, and it also needs to map the
-certificate's distinguished name (DN) to the appropriate `kibana_system` role.
+client certificate's distinguished name (DN) to the appropriate `kibana_system` role.
 
 NOTE: Using a PKI realm is a gold feature. For a comparison of the Elastic license levels, see https://www.elastic.co/subscriptions[the
 subscription page].
 
 To configure {kib} and {es} to use mutual TLS authentication:
 
-. <<using-kibana-with-security,Set up {kib} to work with {security}>> with a username and password.
+. *<<using-kibana-with-security,Set up {kib} to work with {security} with a username and password>>.*
 
-. <<configuring-tls-kib-es,Set up TLS encryption between {kib} and {es}>>. At a minimum, this requires a server certificate for {es}.
+. *<<configuring-tls-kib-es,Set up TLS encryption between {kib} and {es}>>.*
```

Comment:
In this and all other occurrences, the bolded first sentence should not be bolded. If you want it to stand out from the rest of the big block of text, you can perhaps add an extra line break. Otherwise, I think it's diverging too much from the style of all the procedural pages across the library. It might be possible to request this as a stylesheet change across the library, however I think it would be a hard sell given that many procedures contain steps that are much less verbose than these.
```suggestion
. <<configuring-tls-kib-es,Set up TLS encryption between {kib} and {es}>>.
```

Reply:
Ah, alright I'll remove the bold styling. Thanks!